pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,131 @@
+<h2><a href="https://code.org/educate/weblab"><strong>Web Lab</strong></a></h2>
+<div class="col-33" style="padding-right: 20px;">
+    <img src="/images/athome/web-lab.jpg">
+    <p><i>Photo courtesy of Fundación Sadosky</i><p>
+</div>
+<div class="col-66">
+    <p>Web Lab is a programming environment where students can make simple web pages using HTML and CSS. Design your web pages and share your site in seconds. Recommended for students who have completed at least Chapter 1 of <a href="https://curriculum.code.org/csd/unit2/">Web Development</a>.</p>
+    <br>
+    <p><a href="https://studio.code.org/docs/weblab/">Web Lab Documentation</a></p>
+</div>
+<div style="clear: both; margin-bottom: 20px"></div>
+<table>
+    <thead>
+        <tr>
+            <th>Web Lab Project Ideas</th>
+            <th></th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h3><strong>Digital Book Report</strong></h3><p>Use Web Lab to create a presentation for a class assignment. Add text and images and cite your sources.</p><img src="/images/athome/book-report.png" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/1t9MluD6MXBpXC6v-O2iJZth2XngFNt1cNcfDnSnAzMA/edit">More Details</a></p><p><a href="https://docs.google.com/document/d/1b-r2fmXMX5UzQDOZO5Ti40pDhA1_fup232Tlim1bdp4/edit#">Project Guide</a></p></td>
+            <td><h3><strong>Choose Your Own Adventure</strong></h3><p>Write a branching story using hyperlinks and separate pages for each reader decision point.</p><img src="/images/athome/cyoa.png" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/1ERjIca3bFmm1Gj8x5YKNB2bvd3gwTdvaqdq4XUpBxZI/edit#">More Details</a></p><p><a href="https://docs.google.com/document/d/1bMPvQTEtThFx5R0yIoJN4290SerK9lLgtPXp5zS-umQ/edit#">Project Guide</a></p></td>
+            <td><h3><strong>Journal or Blog</strong></h3><p>Create a space for you to write about your experiences. Add a new page each daily or week!</p><img src="/images/athome/journal.png" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/104TYIDCUq8n7AiSEYFMqdkBqes8Yr9WCi1zqEdoBSkU/edit#">More Details</a></p><p><a href="https://docs.google.com/document/d/1bMPvQTEtThFx5R0yIoJN4290SerK9lLgtPXp5zS-umQ/edit#">Project Guide</a></p></td>
+
+        </tr>
+    </tbody>
+</table>
+<div style="clear: both; margin-bottom: 40px"></div>
+<h2><a href="https://code.org/educate/gamelab"><strong>Game Lab</strong></a></h2>
+<div class="col-33" style="padding-right: 20px;">
+    <img src="/images/athome/game-lab.jpg">
+</div>
+<div class="col-66">
+    <p>Game Lab is a programming environment where you can make simple animations and games with objects and characters that interact with each other. Recommended for students who have completed at least Chapter 1 of <a href="https://curriculum.code.org/csd-19/unit3/">Interactive Animations and Games</a>.<p>
```

Comment:
Similar to my previous comment: 
Before: 
`Chapter 1 of <a href="https://curriculum.code.org/csd-19/unit3/">Interactive Animations and Games</a>`

After:
`Chapter 1 of Interactive Animations and Games from <a href="https://studio.code.org/courses/csd">CS Discoveries</a>`

@dancodedotorg

Reply:
Same comment as above

Reply:
Oh - sorry. I also have a "Yes - and" proposal:

After v2:
`Chapter 1 of Interactive Animations and Games from <a href="https://studio.code.org/courses/csd">CS Discoveries</a> or the <a href="https://studio.code.org/s/csd3-virtual">Self-Paced Introduction to Game Lab</a>`